src/Button/index.js

```diff
@@ -73,7 +83,12 @@ const pulsingAnimation = keyframes`
    const color = props.color && props.color[breakpoint]
    const fontFamily = props.fontFamily && props.fontFamily[breakpoint]
    const padding = props.padding && props.padding[breakpoint]
-   const size = props.size && props.size[breakpoint]
+   let size = props.size && props.size[breakpoint]
+
+   // If user passes invalid button 'size', use size 'md' instead
+   if (!['xs', 'sm', 'md', 'lg'].includes(size)) {
```

Comment:
If these possible values change, then we'll have to rely on memory to change this one line. A safer way to do this might be: I'd instead do: 

```js
if (!Object.keys(buttonSizeDefaults).includes(size))
```

Also, I would throw an error instead of default to some value, such as `md`. Why? Because it makes debugging more difficult, because the person (probably me) will try to use `Button` incorrectly, but confused why the size is always medium even though I keep trying incorrect things.

Reply:
First part of this comment was not addressed